### DISCUSSION TITLE
Plotman interactive

### DISCUSSION THREAD

**Discussion by Smange0326** [ericaltendorf/plotman#365] [Q&A]
I get a "attribute error: 'truple' object has no attribute known. Other than that everything seems to work fine. 

I did download the gui too so i could see what was going on jut want to be able to monitor everything via ssh but my plotman status works fine so thatll do for now

**Comment by BasilHorowt** [comment]
Sounds like [1] the `tuple` error was a temporary issue with v0.3 and is fixed in v0.3.1, please try updating :)


1: keybase://chat/chia_network.public#plotman/17493
